modules/spring-config/src/main/resources/META-INF/mule-module.properties

```diff
@@ -0,0 +1,32 @@
+module.name=spring-config
+
+artifact.export.classPackages=org.mule.runtime.config.spring,\
```

Comment:
Can we add semantics to add all packages after .? i.e.: org.mule.runtime.config.spring.dsl.api.**